Resources/Prototypes/_LostParadise/Flavors/flavors.yml

```
@@ -0,0 +1,29 @@
+  flavorType: Complex
+  description: flavor-complex-berry-tea
+
+- type: flavor
+  id: orangetea
+  flavorType: Complex
+  description: flavor-complex-orange-tea
+
+- type: flavor
+  id: flowertea
+  flavorType: Complex
+  description: flavor-complex-flower-tea
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)